src/servers/src/postgres/handler.rs

```diff
@@ -261,6 +282,28 @@ impl ExtendedQueryHandler for PostgresServerHandlerInner {
         self.query_parser.clone()
     }
 
+    async fn on_execute<C>(&self, client: &mut C, message: Execute) -> PgWireResult<()>
+    where
+        C: ClientInfo + ClientPortalStore + Sink<PgWireBackendMessage> + Unpin + Send + Sync,
+        C::PortalStore: PortalStore<Statement = Self::Statement>,
+        C::Error: Debug,
+        PgWireError: From<<C as Sink<PgWireBackendMessage>>::Error>,
+    {
+        let _slow_query_timer = client
+            .portal_store()
+            .get_portal(message.name.as_deref().unwrap_or(DEFAULT_NAME))
+            .and_then(|portal| {
+                self.slow_query_recorder.as_ref().map(|recorder| {
+                    recorder.start(
+                        SlowQuery::Sql(portal.statement.statement.query.clone()),
+                        self.session.new_query_context(),
+                    )
+                })
+            });
+
+        self._on_execute(client, message).await
```

Comment:
It records the elapsed time of `on_execute`, not `_on_execute` itself.

Reply:
I have already removed it since the old logic has been removed.